Finally

UFFR opened the pull request:
![satori dead inside](https://user-images.githubusercontent.com/63076393/124516004-c7bc8180-ddae-11eb-98c2-60512e8a2513.jpg)

UFFR committed:
Cobalt ore and others

UFFR committed:
Cobalt ore related recipes

UFFR committed:
Cinnabar ore, recipes, and minor improvements

UFFR committed:
Missing asset

UFFR committed:
Hopefully fixed some issues

UFFR committed:
Maybe it will work this time

UFFR committed:
Yeah

UFFR committed:
Almost forgot about this one

UFFR committed:
Absolutely everything

Massive changes, hope it doesn't break anything

UFFR committed:
Absolutely everything 2: Tokyo Drift

Yeah

UFFR committed:
Singularity Generator pain

Basis of the beta Singularity Generator (called Watz Super-Compressor in-game, name pending). Still working on the recipes and such, will take a while. Some miscellaneous other things too, but not as significant.

UFFR committed:
Sing Gen and Nuclear Blast Furnace

Singularity Generator and an RTG powered Blast Furnace are in! Still beta, but they have recipes and more or less function.

UFFR committed:
Bit of everything

Mixed back of additions and improvements over existing code and implementations, too long to list.

UFFR committed:
Too many things to list

Don't remember everything that changed, but I did make a lot of text dependent on the lang file so they can be translated now, had the benefit of significantly reducing the amount of giant lengths of if statements in some of the classes.
Additionally, I tried to reduce that sort of thing in the starter kit and grenade classes. The starter kit now receives arrays in its constructor which it uses for the giving to inventory, works most of the time, but a few that used [Arrays.fill] are buggy, working on it. The grenade class also now can take the grenade entity's class as a parameter in the constructor as well, surprisingly bugless.
Oh right, energy guns and a bunch of things that are references, we'll see how they develop.
Remember when this fork was just about supercomputers and storage devices?
Oh, so I did end up listing them...

UFFR committed:
Misc additions

Some misc improvements and additions here and there. Many of them are broken ironically, but Bob can fix those, not like they're main priority to begin with.

HbmMods commented:
![image](https://user-images.githubusercontent.com/16674258/124633291-69ae9d80-de85-11eb-9855-e3c8d905d153.png)

I will thank you in the distant year of 35,924 when I'm done reviewing and merging all these changes.

UFFR committed:
Reworks and additions

Some reworkings/modifications of things, a handful of new additions. A bit janky at the moment.

UFFR committed:
Some more stuff

Mostly making this commit to backup changes. *Slightly* less janky than the previous one though.

UFFR committed:
Commit for backup

Nothing big yet, just a backup. Less buggy, but still a bit buggy.

UFFR committed:
Another backup

I'm putting the FTL reactor on hold, it's not going well, would be easier to start from scratch. Regardless, I'll still be continuing work when possible.

UFFR committed:
Updates to the project

Some weapons, tools, and finally some basic work on the supercomputers.

UFFR committed:
Some bug fixes and other work

Fixed a few bugs, notably with the NumberDisplay class and some of the greener machines.
Other misc work as well, including dangerous experimental code!

UFFR committed:
New gun stuff!

Featuring more experimental code and other tweaks, but who cares about that when we have some new gun stuff? The Benelli M4 had some detail work and a texture done by FrizzleFrazzle, the Lunatic sniper, as well as an AR style 5.56mm gun, so major props to FrizzleFrazzle. Both now use (stolen) sound assets from Project Brutality 3 (also on GitHub if you want to look it up). There are also several new beta gun configurations still in testing attached to the test_gun item. They also use the new .45 ACP and 7.62x51mm NATO (.308 Winchester) cartridges. The Lunatic sniper uses a special 20x155mm cartridge that is very high powered and spicy all around.

UFFR committed:
Some code fixes and other additions

Trying to fix up some of the code while adding more stuff, particularly the custom warheads and cores, which are pretty beta still. Some mechanic changes were also implemented. Also plan to reduce spaghetti in the item classes later.

Ignore the new battery block TEs, they're just jokes really.

UFFR committed:
Backup

Commit mostly for backup. The RTG rework is mostly finished now thought.

UFFR committed:
Backup

Commit mostly for backup. The RTG rework is mostly finished now though.

UFFR committed:
Various additions and reworks

A lot of changes, don't know what else to say. Summary-ish:

I will say that I added a lot of heavier transuranics (not all have to be added, just to see if they'll fit in) as well as U-234 which comes from the decay of Pu-238 RTGs. Also plan on it to come from spent RBMK fuel, but alas I don't have that in my fork so I can't add it.
Changed the custom toxicity handler to use an enumerate since that makes more sense.
Added a "chad" screwdriver since main has the screwdriver break.
Misc. changes to various classes to reduce spaghetti slightly, will probably never be merged but it makes me happier.
Added rounding to the tooltips of radioactive things, useful for when realistic RAD/s is on.
Plans on Cf-252 + Zirconium neutron sources, for the RBMK of course, extremely strong. Maybe I could add it to the breeding reactor's fuel?
Possibly more things, but I don't remember them all!

UFFR committed:
Merge remote-tracking branch 'origin/master'

hhhh
Remind me never to fix a spelling/grammar issue in a previous commit again.

UFFR committed:
Merging with main repo

Suffice to say I found the 10th circle of Hell
This better be worth it Boberto

UFFR committed:
Merge branch 'Pull_attempt'

UFFR committed:
Commit for merge

This better be merged properly Boberto or I swear I'll rewatch KKHTA and find the perfect method to commit [extreme acts of uncivilized ultra-violent behavior] on you.

UFFR commented:
In all seriousness, there are still a few quirks here and there that aren't completely worked out yet that I either just couldn't find or don't know how to fix. Most of them are pretty simple so should be resolved easily. As for machines, items, or other features that are beta stage (which most of them are), I guess you'll either have to quickly finish or disable them. The RTG blast furnace is pretty much done though, the Watz Super-compressor works, but only with "shaped" recipes and it has no model, the Tsukuyomi weapon works, but is unfinished and unbalanced, the TWR is still a placeholder, the divine light weapon only works in flood mode since I couldn't figure out rays, the atomic clock _sort of_ works, but the time system behind it is still a bit new and I don't have any debugging commands for it, the new guns work fine, their animations are rather unsophisticated, but are passable, the energy gun base class seems to work as well as the fast chargers for the moment. Custom cores and warheads are kind of in, but they aren't functional and likely won't be for a while. There may still be a few missing recipes either due to me missing them or simply not having the resources to add them, such as the Niobium superalloy recipe, which requires that alloy forge thing proposed a while back.

UFFR commented:
There are also temporary/placeholder recycling recipes for some depleted fuel types (Schrabidium fuel) but you can change/remove those entirely at your discretion.

HbmMods committed:
Delete .log

fuck around & find out, #1

UFFR committed:
Delete .metadata directory

Seemingly useless folder

UFFR committed:
Test commit

To see if we can merge this stuff properly

HbmMods committed:
Merge branch 'master' into master
